packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -216,12 +217,12 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
   public componentDidMount(): void {
     if (this._comboBoxWrapper.current) {
       // hook up resolving the options if needed on focus
-      this._events.on(this._comboBoxWrapper.current, 'focus', this._onResolveOptions, true);
+      <EventListener target={this._comboBoxWrapper.current} type="focus" listener={this._onResolveOptions} />;
```

Comment:
this doesn't belong here; in render.
also make sure not to drop the capture flag.